examples/basic_cog.py

```diff
@@ -0,0 +1,81 @@
+import discord
+from discord.ext import commands
+
+# https://discordpy.readthedocs.io/en/latest/ext/commands/cogs.html is the documentation page for cogs.
+description = '''An example bot to showcase the discord.ext.commands extension
+module.
+There are a number of utility commands being showcased here.
+The example uses cogs to organize the code
+'''
+
+intents = discord.Intents.default()
+intents.members = True
+
+
+# We define the basic bot, setting the prefix, description and intents
+
+bot = commands.Bot(command_prefix='?',
+                   description=description, intents=intents)
+
+
+# we define the class that will house the Utility commands
+# the class inherits from the commands.Cog class
+# The main advantage is we can take advantage of Object Oriented Programming
+# This also groups all commands in the help command under the `Utility` cog
+class Utility(commands.Cog):
+    """
+    A simple utility cog
+    """
+
+    def __init__(self, bot) -> None:
```

Comment:
```suggestion
    def __init__(self, bot):
```
I don't think type hinting for `__init__` is useful